src/static/css/index.css

```diff
@@ -392,7 +402,8 @@ p {
     margin-top: 80px;
     margin-right: -46px;
   }
-  .intro-image-2019-wrapper {
+  .intro-image-2019-wrapper,
+  .intro-image-2020-wrapper {
```

Comment:
Similarly, `.intro-image-wrapper`

Reply:
As per above this should be part of #894

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/893/commits/b5351ed6b55a2e02432f50f99765910e1c794ae9